update docs to reflect changes in settings page, access tokens in studio (#5064)
iterative/dvc.org: content/docs/studio/user-guide/account-and-billing/index.md

# Account Management

To open your account settings, click on your user icon on the top right corner
of DVC Studio, and go to your `Settings`. You can view and update the following
settings:

- [General settings](#general-settings)
  - [Profile details](#profile-details) update your name and profile picture
  - [Account details](#account-details) manage your username, password, email
    addresses, and delete your account

> - [Login connections(Git integrations)](#login-connections-git-integrations)

Reviewer: ```suggestion
- [Git connections](#login-connections-git-integrations)
```

Since we're renaming this section in Studio.

  with GitHub, GitLab and Bitbucket
- [Cloud connections](#cloud-connections) for data remotes

> - [Manage teams](#manage-teams) that you own

Reviewer: ```suggestion
- [Teams](#manage-teams) that you own
```

</admon>

## General settings

> In your settings page, the general tab includes your profile, account settings.

Reviewer: ```suggestion
In your settings page, the general tab includes your profile and account settings.
```

### Profile details

> You can update your first name, last name and profile picture from profile

Reviewer: ```suggestion
Here you can update your first name, last name and profile picture
```

> section.

Reviewer: ```suggestion
.
```

### Account details

> In the account section, your username is displayed. You can also update your

Reviewer: ```suggestion
Here, you can also update your
```

> username.

Reviewer: ```suggestion
username, 
```

> 

Reviewer: ```suggestion
```

> You can also change your password and email address from this section.

Reviewer: ```suggestion
, password and email addresses.
```

You can delete your non-primary email addresses.

#### Delete account

If you delete your account, all the projects you own and the links that you have
shared will be permanently deleted. So, click on `Delete my account` only if you
are absolutely sure that you do not need those projects or links anymore.

<admon>

Deleting your account in DVC Studio does not delete your Git repositories.

</admon>

> ## Login Connections (Git integrations)

Reviewer: ```suggestion
## Git Connections
```
Let's use either Git Connections or Git Integrations, but not both.

  GitHub app on additional organizations or repositories, or even remove the app
  from organizations or repositories where you no longer need it.

## Cloud connections

In this section, you can view, add and update credentials for cloud resources.
These credentials are used to
[fetch project data from data remotes](/doc/studio/user-guide/experiments/configure-a-project#data-remotes--cloud-storage-credentials).

To add new credentials, click `Add credentials` and select the cloud provider.
Depending on the provider, you will be asked for more details.

![](https://static.iterative.ai/img/studio/s3_remote_settings_v2.png)

The credentials must have the required permissions. For
[data remotes](/doc/studio/user-guide/experiments/configure-a-project#data-remotes--cloud-storage-credentials),
you can find more details about required permissions [here][data remote].

[data remote]: /doc/user-guide/data-management/remote-storage
[settings]: https://studio.iterative.ai/user/_/settings

Finally, click `Save credentials`.

> ## Manage teams

Reviewer: ```suggestion
## Teams
```

> In this section, you can view and manage the teams that you own. You can also

Reviewer: Question: Does this list also include teams where I am not an admin (owner)?

Author: Yes, with a lablel. Updating it to 
`In this section, you can view all the teams you are member of.`

<img width="880" alt="Screenshot 2023-12-21 at 12 36 11" src="https://github.com/iterative/dvc.org/assets/20840228/ae61c68b-57c0-40bd-91d5-5d915dd336e0">

> In this section, you can view and manage the teams that you own. You can also

Reviewer: ```suggestion
In this section, you can view and manage the teams that you own.
```

> create new teams.

Reviewer: ```suggestion

```

> You can click on `select` to switch to the team dashboard. Or, you can click on

Reviewer: ```suggestion
Click on `select` to switch to the team's dashboard. Or, click on
```

`manage` to go to the team settings page and manage the team.

To create a new team, click on `Create a team` and enter the team name. You can

> invite members to the team by entering their email addresses.

Reviewer: ```suggestion
invite members to the team by entering their email addresses. Find more details [here](/doc/studio/user-guide/team-collaboration#create-a-team).
```

## Tokens

> 

Reviewer: Section on `Studio access token` is missing.

Author: I am trying to incorporate https://github.com/iterative/dvc.org/issues/5052 this as well.
